docs/user-guide/configuring-zowe-via-jcl.md

```diff
@@ -0,0 +1,62 @@
+ # Configuring Zowe via JCL
+
+One option to configure Zowe is by directly customizing JCLs. The Zowe Runtime Dataset `SZWESAMP` contains JCL samples that have templates referencing Zowe YAML parameters. These samples should not be submitted without modification. Samples that are submitted without modification will end unsuccessfully with a JCL ERROR status.
+
+Edit and submit the job `SZWESAMP(ZWEGENER)` to validate the contents of your `zowe.yaml` before resolving the `JCL templates` and placing the resulting JCL into a separate data set created during installation. The location is specified in `zowe.setup.dataset.jcllib`.
```

Comment:
`jcllib` is not created by `zwe install`. This dataset is created by `zwe init (generate)`:
_...into a separate data set created by `ZWEGENER` job._